src/styles/BookForm.css

```diff
@@ -1,29 +1,74 @@
 .containerCreateBooks {
   width: 90%;
-  height: 100px;
+  height: 150px;
   margin: 80px auto;
   padding: 20px 0;
+  padding-bottom: 60px;
+  margin-top: 10px;
 }
 
 form {
-  display: flex;
-  justify-content: space-between;
+  font-family: 'Montserrat', sans-serif;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+}
+
+.line {
+  height: 0.125rem;
   margin-top: 10px;
-  box-sizing: border-box;
-  border-radius: 12px;
+  margin-bottom: 30px;
+  border: solid 1px #ccc;
+}
+
+h3 {
+  width: 11rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-size: 1.25rem;
+  font-weight: bold;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.add-btn {
+  width: 11.5rem;
+  height: 2.813rem;
+  border-radius: 3px;
+  background-color: #0290ff;
   border: none;
+  color: #fff;
 }
 
 input {
-  width: 25%;
-  height: 40px;
+  width: 14.813rem;
+  height: 2.588rem;
+  border-radius: 4px;
+  opacity: 0.8;
+  font-size: 1rem;
+  border: solid 1px #ccc;
+  padding-left: 1.063rem;
 }
 
-.btn {
-  width: 20%;
-  height: 40px;
+select {
+  width: 14.813rem;
+  height: 2.588rem;
+  opacity: 0.5;
+  padding: 0.375rem 0.938rem 0.563rem 1rem;
+  border-radius: 4px;
+  border: solid 1px #ccc;
+}
+
+.book-form {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10px;
+  box-sizing: border-box;
+  border-radius: 12px;
+  border: none;
 }
```

Comment:
- [OPTIONAL] I think it would be better to increase the width of the first input according to what we have in the given given. You can do this by adding `flex: 1` only to the `title input` and adding some gap, let's say `gap: 20px` to the `.book-form`

Reply:
Thank you for your suggestion. This makes my project look like the original design.